libnd4j/include/ops/declarable/platform/vednn/veda_helper.cpp

```diff
@@ -0,0 +1,42 @@
+#include "veda_helper.h"
+
+// https://github.com/SX-Aurora/veda/issues/16
+// to solve the issue related to graceful shutdown, above, we will use ThreadLocalScopVeda and SCOPED_VEDA_CONTEXT
+struct ThreadLocalScopVeda {
+  bool isOk = false;
+
+  ThreadLocalScopVeda() = default;
+
```

Comment:
We'll need to tie this to a device at at the java level as well I believe? Do you have any plans to merge the 2 concepts?